.github/workflows/ci-joli.yaml

```diff
@@ -63,3 +63,10 @@ jobs:
 
       - name: Run tests
         uses: julia-actions/julia-runtest@latest
+
+      - name: Run examples
+        working-directory: examples
+        run: |
+          julia -e 'using Pkg; Pkg.add("JOLI")'
+          julia -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()))'
```

Comment:
That's adding JOLI twice, once in `add` and once in `dev`. JOLi already instantiated an simple `julia --project` should be enough